aws/dubbd-aws/zarf.yaml

```
@@ -60,6 +62,14 @@ components:
             mute: true
             setVariables:
               - name: LOKI_FORCE_DESTROY
+          - cmd: |
+              if [ -z "${ZARF_VAR_VELERO_FORCE_DESTROY}" ]; then
+                echo "${ZARF_VAR_EPHEMERAL}"
+              else
+                echo "${ZARF_VAR_VELERO_FORCE_DESTROY}"
+              fi
+            setVariables:
```

nit: preferable to set mute here for consistency and cleaner output.

```suggestion
            mute: true
            setVariables:
```

---

Fixed